File: Simplex.Core/Channels/IChannel.cs
namespace Tmon.Simplex.Channels
{
    /// <summary>
    /// 작업단위(액션)을 발행하거나, 구독할 때 사용될 채널 
    /// </summary>
    public interface IChannel 
    {
        /// <summary>
        /// 채널의 식별자
        /// </summary>
        string Id { get; }
    }
}

File: simplex.java/src/main/java/kr/co/tmon/simplex/channels/IChannel.java
package kr.co.tmon.simplex.channels;

/**
 * 작업단위(액션)을 발행하거나, 구독할 때 사용될 채널 
 * @author yookjy
 *
 */
public interface IChannel {
	
	/**
	 * 채널의 고유 식별자를 조회합니다.
	 * @return 식별자
	 */
	public String getId(); 
}
